### PR TITLE
chore: sanitize payload before sending to AS

### PR DESCRIPTION
**Pull request by ashishmax31**

### CODE REVIEW

File: managedtenants/utils/ocm.py [hunk 1]
```diff
@@ -130,6 +130,19 @@ def _sanitize_addon_metadata_for_addons_service(self, addon, metadata):
 
         return addon
 
+    def _sanitize_addon_imageset_for_addons_service(self, addon, imageset, metadata):
+        mapped_key = self.IMAGESET_KEYS["addOnParameters"]
+        # This will remove order attribute in addon.
+        if imageset.get("addOnParameters") is None:
+            addon[mapped_key] = metadata.get("addOnParameters", [])
+        else:
+            addon[mapped_key] = imageset.get("addOnParameters", [])
```
Comment (yashvardhan-kukreja) [inline managedtenants/utils/ocm.py:139]: but we just wanna remove the `order` attribute in each element under addOnParameters, not the addonParameteres array itself :/

Comment (ashishmax31) [reply managedtenants/utils/ocm.py:139]: no we are not removing them, we are just removing the extra "items" attribute and using the stuff from the metadata/imageset. (Which don't have an order attribute)

Comment (ashishmax31) [reply managedtenants/utils/ocm.py:139]: This is how its already being done: https://github.com/mt-sre/managed-tenants-cli/blob/f85b9f5ac7b216a2d976a6f203afce353a348c31/managedtenants/utils/ocm.py#L126

Comment (yashvardhan-kukreja) [reply managedtenants/utils/ocm.py:139]: ohh nvm, I went crazy for a bit and misread the `.get(.., <default-value>)` syntax as `addon[mapped_key] = []` 

(feel like Friday)
:joysob:

**Review by yashvardhan-kukreja — APPROVED** [review]
/lgtm

**Pull request merged**